cmd/tracee-rules/output_test.go

```diff
@@ -250,3 +250,58 @@ HostName: foobar.local
 
 	}
 }
+
+func TestJsonTemplateV1(t *testing.T) {
+	testCases := []struct {
+		testName     string
+		finding      detect.Finding
+		expectedJson string
+	}{
+		{
+			testName: "convert finding to v1",
+			finding: detect.Finding{
+				Data: map[string]interface{}{
+					"a": 123,
+					"b": "c",
+					"d": true,
+					"f": map[string]string{
+						"123": "456",
+						"foo": "bar",
+					},
+				},
+				Event: trace.Event{
+					ProcessID:   21312,
+					Timestamp:   1321321,
+					UserID:      0,
+					ContainerID: "abbc123",
+					EventName:   "execve",
+				}.ToProtocol(),
+				SigMetadata: detect.SignatureMetadata{
+					ID:          "TRC-1",
+					Version:     "0.1.0",
+					Name:        "Standard Input/Output Over Socket",
+					Description: "Redirection of process's standard input/output to socket",
+					Tags:        []string{"linux", "container"},
+					Properties: map[string]interface{}{
+						"Severity":     3,
+						"MITRE ATT&CK": "Persistence: Server Software Component",
+					},
+				},
+			},
+			expectedJson: "{\"Data\":{\"a\":123,\"b\":\"c\",\"d\":true,\"f\":{\"123\":\"456\",\"foo\":\"bar\"}},\"Context\":{\"timestamp\":1321321,\"processorId\":0,\"processId\":21312,\"threadId\":0,\"parentProcessId\":0,\"hostProcessId\":0,\"hostThreadId\":0,\"hostParentProcessId\":0,\"userId\":0,\"mountNamespace\":0,\"pidNamespace\":0,\"processName\":\"\",\"hostName\":\"\",\"containerId\":\"abbc123\",\"eventId\":\"0\",\"eventName\":\"execve\",\"argsNum\":0,\"returnValue\":0,\"stackAddresses\":null,\"args\":null},\"SigMetadata\":{\"ID\":\"TRC-1\",\"Version\":\"0.1.0\",\"Name\":\"Standard Input/Output Over Socket\",\"Description\":\"Redirection of process's standard input/output to socket\",\"Tags\":[\"linux\",\"container\"],\"Properties\":{\"MITRE ATT&CK\":\"Persistence: Server Software Component\",\"Severity\":3}}}",
```

Comment:
I believe this can be more readable with Go multiline strings, wdyt?

```suggestion
			expectedJson: `{
	"Data": {
		"a": 123
	}
}`,
```

Reply:
Done